crates/solver/src/settlement/external_prices.rs

```diff
@@ -16,7 +16,7 @@ use std::collections::{BTreeMap, HashMap};
 /// A collection of external prices used for converting token amounts to native
 /// assets.
 #[derive(Clone, Debug)]
-pub struct ExternalPrices(HashMap<H160, BigRational>);
+pub struct ExternalPrices(pub HashMap<H160, BigRational>);
```

Comment:
@nlordell made this private intentionally so users of the struct wouldn't have to / be able to see the internals of it. It would be nicer if we added a new public method in the impl below to get the price with a comment explaining in what unit the price is.